examples/alloy-editor/app.js

```diff
@@ -1,25 +1,29 @@
-import widget from '../../lib/api'
+import { init } from '../../lib/api'
 
-let AlloyEditor = window.AlloyEditor
-widget.init((api) => {
-  api.window.updateHeight(200)
-  api.window.startAutoResizer()
-  let editor = AlloyEditor.editable('content')._editor
+const AlloyEditor = window.AlloyEditor
+
+init((widget) => {
   let currentValue
-  editor.setData(api.field.getValue())
 
-  api.field.onValueChanged((value) => {
+  widget.window.updateHeight(200)
```

Comment:
Why set the height explicitly if in the next line you use the auto resize option?